scvi/model/base/_vaemixin.py

```diff
@@ -118,13 +123,15 @@ def get_reconstruction_error(
         return reconstruction_error
 
     @torch.no_grad()
+    @unsupported_in_latent_mode
```

Comment:
Based on my proposed changes, this would be supported in latent mode because inference would work just fine.